src/app/public/modules/modal/modal.component.ts

```diff
@@ -174,7 +174,7 @@ export class SkyModalComponent implements AfterViewInit {
     this.hostService.onOpenHelp(this.helpKey);
   }
 
-  public closeButtonClick() {
+  public close() {
```

Comment:
This is a public method, so we can't rename it. If you like the name, "close" for semantic reasons, you can add another method that does the same thing (one method for the public API (`close`), and one for the button event listener (`closeButtonClick`)).

Reply:
done